languages/v/brackets.scm

```diff
@@ -0,0 +1,5 @@
+("(" @open ")" @close)
+("[" @open "]" @close)
+("{" @open "}" @close)
+
+(interpolation "${" @open "}" @close)
```

Comment:
```suggestion
(string_interpolation (interpolation_opening) @open (interpolation_closing) @close)
```

Reply:
Read the grammar.js file and see where the query comes from now